src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/** Deadline form of Task */
```

Comment:
Should this be a multi-line comment instead? The coding standard only states "Javadoc of class members can be specified on a single line". However, since this comment is for a class, perhaps it should follow the general form for Javadoc comments where "The opening `/**` [is] on a separate line"?